frontend/packages/dev-console/src/components/monitoring/overview/MonitoringOverviewAlerts.tsx

```diff
@@ -48,7 +48,7 @@ const MonitoringOverviewAlerts: React.FC<MonitoringOverviewAlertsProps> = ({ ale
             {message}
             <div className="odc-monitoring-overview-alerts__timestamp">
               <span className="pf-v6-u-font-size-xs pf-v6-u-text-color-subtle">
-                {fromNow(activeAt)}
+                {fromNow(activeAt ?? new Date())}
```

Comment:
probably don't want to show that an alert was "just now" as a fallback

```suggestion
                {activeAt ? fromNow(activeAt) : DASH}
```